src/pages/documentation/platform-assurance/tutorials/optimize-configuration-view.md

```diff
@@ -0,0 +1,35 @@
+# Optimize Setup Validation view
+
+## Overview
+
+You can use the Optimize Setup Validation view to check if your application has been set up successfully to use Adobe Experience Platform Optimize. The view validates all pre-requisites have been met, and offers feedback on how to fix your setup in cases where requirements have been missed.
+
+## Getting Started
+
+To use this view, follow the steps outlined below:
+
+1. [Set up Adobe Experience Platform Assurance](../set-up.md)
+2. [Create](./index.md#creating-sessions) and [connect](./index.md#connecting-to-a-session) to an Assurance session
+3. In the **Home** view, select **Validate Setup**
+
+![Validate Setup](./assets/optimize-configuration/optimize-setup-view.png)
+
+## Datastream Setup Validation
+
+At the top of the Optimize Setup Validation view, Platform Assurance inspects your current application's datastream to see if Target or Offer Decisioning has been enabled. In order to use optimize, at least one of extensions needs to be enabled inside your datastream.
```

Comment:
I _believe_ the secondary usage of Assurance is just Assurance by itself, we can just say "... view, Assurance inspects your..."